wgpu-core/platform-features/windows-linux-android-emscripten/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "wgpu-core-platform-windows-linux-android-emscripten"
```

Comment:
for all proxy crates: This crate includes `hal` not `core`, so it's incorrectly named right now
```suggestion
name = "wgpu-hal-platform-windows-linux-android-emscripten"
```

Reply:
I was thinking about it as "wgpu-core's dependencies on this platform", but if you want this switched, not particularly married to it.

Reply:
`wgpu-core-deps-platform-windows-linux-android-emscripten` would work for me then 😄 
otherwise it is "the hal dependency for X" imho

Reply:
How about dropping the platform `wgpu-core-deps-windows-linux-android-emscripten`

Reply:
like! :)

Reply:
Done.